main.py

```diff
@@ -200,9 +201,11 @@
   ('/cron/schema_migration_comment_activity', schema_migration.MigrateCommentsToActivities),
   ('/cron/schema_migration_entities', schema_migration.MigrateEntities),
   ('/cron/write_standard_maturity', deprecate_field.WriteStandardMaturityHandler),
+  ('/cron/reindex_all', search_fulltext.ReindexAllFeatures),
 
-  ('/tasks/email-subscribers', notifier.FeatureChangeHandler),
+  ('/admin/find_stop_words', search_fulltext.FindStopWords),
```

Comment:
`/admin/find_stop_words` requires the cron header - does that affect executing this?

Reply:
I discovered that require_cron_header() allows the handler to run if the user is a site admin.  So, I am basically using that to prevent random visitors from hitting that URL to burn our CPU time.   It also means that the various migration scripts that we want to run in GAE don't need cron.yaml entries to be run, so long as you are a site admin.  In fact, maybe their URLs should start with /admin.